src/common/AdaptiveTree.cpp

```diff
@@ -319,18 +318,16 @@ namespace adaptive
   {
     size_t paramPos = url.find('?');
     base_url_ = (paramPos == std::string::npos) ? url : url.substr(0, paramPos);
-    if (paramPos != std::string::npos)
-      manifest_parameter_= url.substr(paramPos);
 
     paramPos = base_url_.find_last_of('/', base_url_.length());
     if (paramPos == std::string::npos)
     {
-      Log(LOGLEVEL_ERROR, "Invalid mpdURL: / expected (%s)", manifest_url_.c_str());
+      Log(LOGLEVEL_ERROR, "Invalid url: / expected (%s)", url.c_str());
       return false;
     }
     base_url_.resize(paramPos + 1);
 
-    paramPos = base_url_.find("://");
+    paramPos = base_url_.find("://", 0);
```

Comment:
You mentioned this change is for consistency with how it is used elsewhere. I think it may be better to go the other way though, the default argument for pos is 0 so specifying it seems redundant.

Reply:
I agree @glennguy, if anything the other uses should have the `, 0` removed. Best to use the default usage when ever possible.

Reply:
Sounds good to me. I'll change the others as well and update PR :)

Reply:
Done :)